docs/core/tutorials/cli-create-console-app.md

```diff
@@ -58,17 +58,21 @@ Let's do a quick walkthrough:
 
     [dotnet run](../tools/dotnet-run.md) calls [dotnet build](../tools/dotnet-build.md) to ensure that the build targets have been built, and then calls `dotnet <assembly.dll>` to run the target application.
     
-    ```console
+    ```dotnetcli
     dotnet run
+    ```
 
+    ```console
```

Comment:
If you break these two, then I think it'd be good to add a sentence in between to indicate what the new code block is showing. You've done that in line 103.